page-changelog.php

```diff
@@ -3,6 +3,80 @@
 </div>
 <p>We recommend you follow our <a href="/release-channels">Release Channels</a> to always have the latest ownCloud Server version which fits your needs.</p>
 
+<h3>Version 8.2.1 <small>November 17 2015</small></h3>
+<ul>
+<li>Sharing improvements:</li>
+<ul>
```

Comment:
Looks like this should be a /ul tag to end the ul - and then all the other ul tag pairs seem to be out-of-sync.